tarantool_test.go

```diff
@@ -3835,7 +3835,7 @@ func TestConnection_NewWatcher_concurrent(t *testing.T) {
 	var wg sync.WaitGroup
 	wg.Add(testConcurrency)
 
-	var ret error
+	errors := make(chan error, testConcurrency)
```

Comment:
Does we had race condition in test scenario or connector implementation itself?

Reply:
In the test we had a shared variable `ret` and set a value for the variable from different goroutines without protection - got data race.